content/v1.14/cli/command-reference.md

```diff
@@ -0,0 +1,416 @@
+[Upbound Marketplace](https://marketplace.upbound.io/).
+
+For example, to install version 0.42.0 of the 
+[AWS S3 provider](https://marketplace.upbound.io/providers/upbound/provider-aws-s3/v0.42.0):
+
+`crossplane xpkg install provider xpkg.upbound.io/upbound/provider-aws-s3:v0.42.0`
+
+#### Flags
+{{< table "table table-sm table-striped">}}
+| Short flag   | Long flag                                        | Description                                                                                     |
+| ------------ | -------------                                    | ------------------------------                                                                  |
+|              | `--runtime-config=<runtime config name>`         | Install the package with a runtime configuration.                                               |
+| `-m`         | `--manual-activation`                            | Set the `revisionActiviationPolicy` to `Manual`.                                                |
+|              | `--package-pull-secrets=<list of secrets>`       | A comma-seperated list of Kubernetes secrets to use for authenticating to the package registry. |
+| `-r`         | `--revision-history-limit=<number of revisions>` | Set the `revisionHistoryLimit`. Defaults to `1`.                                                |
+| `-w`         | `--wait=<number of seconds>`                     | Number of seconds to wait for a package to install.                                             |
+
+{{< /table >}}
+
+#### Wait for package install
+
+When installing a package the `crossplane xpkg install` command doesn't wait for
+the package to download and install. View any download or installation problems 
+by inspecting the `configuration` with `kubectl describe configuration`.
+
+Use `--wait` to have the `crossplane xpkg install` command to wait for a 
+package to have the condition `HEALTHY` before continuing. The command 
+returns an error if the `wait` time expires before the package is `HEALTHY`.
+
+#### Require manual package activation
+
+Set the package to require 
+[manual activation]({{<ref "../concepts/packages#upgrade-policy" >}}), 
+preventing an automatic upgrade of a package with `--manual-activation`
+
+#### Authenticate to a private registry
+
+To authenticate to a private package registry use `--package-pull-secrets` and 
+provide a list of Kubernetes Secret objects. 
+
+{{<hint "important" >}}
+The secrets must be in the same namespace as the Crossplane pod. 
+{{< /hint >}}
+
+#### Customize the number of stored package versions
+
+By default Crossplane only stores a single inactive package in the local package
+cache. 
+
+Store more inactive copies of a package with `--revision-history-limit`. 
+
+Read more about 
+[package revisions]({{< ref "../concepts/packages#configuration-revisions" >}}) 
+in the package documentation. 
+
+### xpkg login
+
+Use `xpkg login` to authenticate to the 
```

Comment:
```suggestion
Use `xpkg login` to authenticate to xpkg.upbound.io, the 
```

I think it's worth explicitly including the domain name. It's going to appear in a lot of places (e.g. as a prefix in any package, like xpkg.upbound.io/crossplane/contrib/foo:v1.0.0). I suspect a lot of Crossplane users will end up being exposed to xpkg.upbound.io as a place where packages are pulled/pushed vs "the Upbound Marketplace".

Reply:
added the domain anywhere there's a Marketplace reference and the domain wasn't there.